joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/frontend/XTypeRecovery.scala

```diff
@@ -1121,8 +1121,10 @@ abstract class RecoverForXCompilationUnit[CompilationUnitType <: AstNode](
       Seq.empty
     )).iterator
 
-    def getKnownTypes: Set[String] =
-      x.allTypes.filterNot(_.toUpperCase.matches("(UNKNOWN|ANY)")).toSet
+    def getKnownTypes: Set[String] = {
+      val all = x.allTypes.filterNot(_.toUpperCase.matches("(UNKNOWN|ANY)")).toSet
```

Comment:
Also seems unnecessary

Reply:
Yes, that was for debugging, forgot to remove, will remove